base/js/admin.js

```diff
@@ -542,7 +586,7 @@ var sowbForms = window.sowbForms || {};
 			});
 			var repeaterHtml = repeaterObject.html().replace(/_id_/g, $nextIndex);
 
-			var readonly = typeof $el.attr('readonly') != 'undefined';
+			var readonly = typeof $el.attr('readonly') !== 'undefined';
```

Comment:
@AlexGStapleton aah, yeah `isUndefined` is probably what we should be using then. We can save that for a refactor later on.